src/scanner.ts

```diff
@@ -505,8 +505,8 @@ export class Scanner {
         }
 
         if (this.escapedChars && type !== Token.Identifier) {
-        	let col: number = ( start - this.lineStart ) + 1;
-            this.errorHandler.throwError(start, this.lineNumber, col, "Keyword must not contain escaped characters");
+            let col: number = ( start - this.lineStart ) + 1;
```

Comment:
Too much spaces around, see the rest of the source file for the example on how to format it.